srcpkgs/teckit/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'teckit'
+pkgname=teckit
+version=2.5.8
+revision=1
+build_style=gnu-configure
+hostmakedepends="libtool autoconf automake"
+makedepends="zlib-devel"
+depends="libteckit"
+short_desc="A Text Encoding Conversion toolkit"
+maintainer="svenper <svenper@tuta.io>"
+license="custom"
```

Comment:
find true license

Reply:
it uses *either* CPL or GPL, should they be separated by a comma?